### PR TITLE
RSpecs working in the blog. 6/6 specs succeeded. (Including model and…

### PR DESCRIPTION
**Pull request by osvaldoFernandez92**
Summary 
Controller specs created. Model specs created.
6/6 specs succeeded
Trello card solved: https://trello.com/c/vupn00WM

NOTE: Unable to perform "redirection when not logged in" spec, because devise won't recognize locales in the URL.

### CODE REVIEW

**Comment by osvaldoFernandez92** [comment]
@epintos

File: Gemfile [hunk 1]
```diff
@@ -15,6 +15,19 @@ gem "twitter-bootstrap-rails"
 
 gem 'devise'
 gem 'rubocop'
+gem 'rb-readline'
+
+group :development, :test do
+  gem 'rspec-rails', '~> 3.0'
```
Comment (epintos) [inline Gemfile:21]: Don't change it now. But take into account that you adding rspec-rails, for example, to the development group and thats not neccesaary

File: spec/factories/articles.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :article do	
+    title "primer blog"
```
Comment (epintos) [inline spec/factories/articles.rb:3]: Here you can use Faker in the future

**Comment by epintos** [comment]
@osvaldoFernandez92 Look at my comments. It good to merge anyway :+1:

**Pull request merged**